diagnostics/precip_buoy_diag/doc/precip_buoy_diag.rst

```diff
@@ -0,0 +1,63 @@
+Precipitation Buoyancy Diagnostic Package
```

Comment:
MDTF_Documentation_precipitation_buoyancy.pdf may be deleted if it contains all the info in this file (see issue #131)

Reply:
Removed pdf documentation https://github.com/ahmedfiaz/MDTF-diagnostics/commit/aa555fdaaf8acb1a4d3ec52fed629636f610367e